pacfetcher_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 func pacjsHandler(pacjs string) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) { w.Write([]byte(pacjs)) }
+	return func(w http.ResponseWriter, r *http.Request) { w.Write([]byte(pacjs)) } // nolint:errcheck
```

Comment:
That link says the Go convention is to not put a space after the comment marker for machine readable comments. That is clearly wrong because godoc comments are machine readable and always have the space, but even build switches in Go do not have it: https://github.com/golang/go/blob/9f78aa77443389d61a3ef910a1c3d4628999d82c/src/runtime/mmap.go#L5

I might be more willing to accept what the golangci lint docs say if they provided a reference or source for their assertion, but without it all I can say is that I have never seen the no-space comment thing anywhere.

I'm going to stick to the much nicer slash-slash-space style until I see something otherwise.

Reply:
go generate requires it. https://blog.golang.org/generate

Reply:
I started arguing, but bah - I don't care enough.

Done

Reply:
Fixed in fc7715bf8dbb439e996234d0949c385ac6b3ce0c and 0fac3dda2d8e3bfc6c9c2d802b37cc4fa9ac560c. Will rebase-squash prior to merging.

Reply:
Fixed in fc7715bf8dbb439e996234d0949c385ac6b3ce0c and 0fac3dda2d8e3bfc6c9c2d802b37cc4fa9ac560c. Will rebase-squash prior to merging.